lib/index.js

```diff
@@ -66,30 +73,58 @@ function ballBoundries(object) {
   }
 }
 
-
 function createFirstRow(num) {
   this.x = -50
   for (var i = 0; i < num; i++) {
-    blocks.push(new Block ({context: context, x: x+= 60}))
+    var block = new Block ({context: context, x: x += 60})
+    blocks.push(block)
   }
 }
 function createSecondRow(num) {
   this.x = -50
   for (var i = 0; i < num; i++) {
-    blocks.push(new Block ({context: context, x: x+= 60, y: 30}))
+    var block = new Block ({context: context, x: x+= 60, y: 30})
+    blocks.push(block)
   }
 }
 function createThirdRow(num) {
   this.x = -50
   for (var i = 0; i < num; i++) {
-    blocks.push(new Block ({context: context, x: x+= 100, y: 50}))
-  }
+    var block = new Block ({context: context, x: x+= 100, y: 50, counter: 3})
+    blocks.push(block)
+ }
 }
 function createFourthRow(num) {
   this.x = -50
   for (var i = 0; i < num; i++) {
-    blocks.push(new Block ({context: context, x: x+= 60, y: 70}))
+    var block = new Block ({context: context, x: x+= 60, y: 70});
+    blocks.push(block)
+  }
+}
+
+function resetBall() {
+  ball.x = canvas.width / 2;
+  ball.y = canvas.height / 2;
+  trigger = false;
+}
+
+function startBall() {
+  if (trigger === true) {
+    ball.speed = 5;
+    ball.move();
   }
+};
+
+function gameOverDisplay() {
+  var gameOverImg = $('.game-over');
+  gameOverImg.css('display', 'block')
+  $('.start-new-game').on('click', function () {
```

Comment:
I noticed we changed the class in the html for this button. We need to change this class to "start-game" instead of "start-new-game